### PR TITLE
add aws-ecr-creds-helper

### PR DESCRIPTION
**Pull request by defo89**

### CODE REVIEW

File: system/aws-ecr-creds-helper/templates/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+spec:
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    metadata:
+      name: ecr-creds-helper
+    spec:
+      backoffLimit: 3
+      template:
+        metadata:
+          name: ecr-creds-helper
```
Comment (databus23) [inline system/aws-ecr-creds-helper/templates/cronjob.yaml:14]: If we add a label to the pod spec (e.g. `cronjob=aws-ecr-creds-helper`) we can query more easily in prometheus. if we do our own alert rule we might as well make it easier for us

Comment (databus23) [reply system/aws-ecr-creds-helper/templates/cronjob.yaml:14]: hmm or maybe this needs to go as a label on the `jobTemplate` above...

File: system/aws-ecr-creds-helper/alerts/ecr-creds-helper.alerts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+groups:
+- name: ecr-creds-helper.alerts
+  rules:
+
+    ### ecr-creds-helper ###
+
+    - alert: EcrCredsHelperJobFailed
+      expr: sum(kube_job_status_failed{job_name=~"aws-ecr-creds-helper-.+"}) by (job_name) > 2
```
Comment (databus23) [inline system/aws-ecr-creds-helper/alerts/ecr-creds-helper.alerts:8]: summarizing by `job_name` is not correct as every job created by the cronjob has its own name. I think we want to summarise over multiple cronjob runs.
Maybe its also better to turn the query around and alert if there are no successful runs in the recent past.
E.g.:
```
sum(increase(kube_job_status_succeeded{job_name=~"aws-ecr-creds-helper-.+"}[8h])) == 0 
```

File: system/aws-ecr-creds-helper/templates/cronjob.yaml (hunk 1 above)
Comment (defo89) [reply system/aws-ecr-creds-helper/templates/cronjob.yaml]: I will keep them added for the sake of completeness although as we found out they still do not show up under job in Prometheus

**Pull request merged**